sdk/samples/iot/iot_samples_common.h

```diff
@@ -14,7 +14,7 @@
 #include <azure/core/az_span.h>
 
 #define SAS_KEY_DURATION_TIME_DIGITS 4
-
+#define ISO_SPEC_TIME_FORMAT "%Y-%m-%dT%H:%M:%S%z" // ISO8601 Time Format

```

Comment:
Is there a way to support fractional seconds? `"yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'FFFFFFF"`

Reply:
I haven't checked, but I don't think doing so will add much value to the sample.